semgrep-core/bin/Main.ml

```diff
@@ -975,6 +975,8 @@ let options () =
     (*e: [[Main_semgrep_core.options]] other cases *)
     "-use_parsing_cache", Arg.Set_string use_parsing_cache,
     " <dir> save and use parsed ASTs in a cache at given directory. Caller responsiblity to clear cache";
+    "-no_filter_rules_regexp", Arg.Clear Flag.filter_rules_regexp,
```

Comment:
Can we invert this flag?

IMO this performance optimization could result in a surprising behavior, and I can anticipate it being quite difficult for users to debug.

Since most users are likely to not encounter a serious performance issue, I'd rather default filtering to `off`, then recommend turning this on in our performance troubleshooting documentation.

cc. @ievans @dlukeomalley for additional thoughts.

Reply:
ok, I'll invert the flag for now.